catalyst-gateway/bin/src/db/index/queries/rbac/get_catalyst_id_from_public_key.rs

```diff
@@ -49,12 +49,12 @@ static VOLATILE_CACHE: LazyLock<Cache<VerifyingKey, QueryResult>> = LazyLock::ne
 });
 
 /// A result of query execution.
-#[allow(dead_code)]
 #[derive(Debug, Clone)]
 pub struct QueryResult {
     /// A Catalyst ID.
     pub catalyst_id: CatalystId,
     /// A slot number.
+    #[allow(dead_code)]
     pub slot_no: Slot,
```

Comment:
If we dont use this field, maybe just remove it ?

Reply:
Done.